docs.md

```diff
@@ -182,10 +182,12 @@ This is to prevent privilege escalation.
 
 Users cannot create ProjectRoleTemplateBindings that violate the following constraints:
 
+- The projectName field must have the format `cluster-id:project-id`, and `project-id` must match the namespace of the
```

Comment:
Might be too late but why aren't we using something like:

```yaml
projectRef:
  projectId: "project-id"
  clusterId: "cluster-id"
```

which is a similar pattern in other objects like:
- `ClusterRoleBinding.roleRef`
- secretRef / configMapRef in Pods

Seems like it would be easier to forget that projectName is really two fields.

Reply:
That might make more sense overall, but is subject to prior discussions on migrating resource fields, so I would not recommend attempting to address it here.